src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -273,6 +274,16 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
     });
   }
 
+  private _handleSeek(e: MouseEvent): void {
```

Comment:
Check for `SUPPORT_SEEK` <https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/media_player/const.py#L50>